From f88143c052a5fb03798088b095935169ec54e04c Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Tue, 28 May 2013 13:38:20 +0800
Subject: [PATCH] ENGR00264465-1 arm: dts: imx6qdl-sabresd:Add LVDS CABC
 pinmux

This patch adds pinmux entries for LVDS0 and LVDS1 to control the
Hannstar LVDS panel CABC function. Pin NANDF_CS2 and pin NANDF_CS3
are configured to be gpio so that they can control the CABC function.

Conflicts:
	arch/arm/boot/dts/imx6qdl-sabresd.dtsi

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit e5607e333915ddd8650af02895979fde3a82197d)
(cherry picked from commit ad9488a6a85053866cc5898923e3e670e901dc68)
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 802b3da..7e93c3c 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -552,6 +552,8 @@
 				MX6QDL_PAD_GPIO_1__WDOG2_B 0x80000000
 				MX6QDL_PAD_NANDF_CS0__GPIO6_IO11 0x80000000
 				MX6QDL_PAD_NANDF_CS1__GPIO6_IO14 0x80000000
+				MX6QDL_PAD_NANDF_CS2__GPIO6_IO15 0x80000000
+				MX6QDL_PAD_NANDF_CS3__GPIO6_IO16 0x80000000
 			>;
 		};
 
-- 
1.8.0

